### PR TITLE
chore(comments): Comments page handler works again

### PR DESCRIPTION
**Pull request by mrclay**
Was broken during resource views refactoring

### CODE REVIEW

**Comment by juho-jaakkola** [comment]
What does this fix? It seems to just rename the variable.

**Comment by mrclay** [comment]
Look closer :) The comments/ URLs didn't work at all.

**Comment by beck24** [comment]
I'm not seeing it either...

File: engine/lib/comments.php [hunk 1]
```diff
@@ -31,23 +31,23 @@ function _elgg_comments_init() {
...
  *
- * @param array $page
+ * @param array $segments
  * @return bool
  * @access private
  */
-function _elgg_comments_page_handler($page) {
+function _elgg_comments_page_handler($segments) {
 
-	$page = elgg_extract(0, $page);
```
Comment (mrclay) [inline engine/lib/comments.php]: Easier to see in split view. This line used to redefine $page, now it creates a new var $page so we still have access to URL segments

Comment (beck24) [inline engine/lib/comments.php]: ah, sneaky, yeah I see it now

**Comment by beck24** [comment]
LGTM

**Pull request merged**